cmd/corectl/main.go

```diff
@@ -200,25 +200,39 @@ func createToken(client *rpc.Client, args []string) {
 	}
 	flags.Parse(args)
 	args = flags.Args()
-	if len(args) < 1 {
+	if len(args) == 2 && *flagNet || len(args) < 1 || len(args) > 2 {
 		fatalln(usage)
 	}
 
-	req := struct {
-		ID, Type string
-	}{
-		ID:   args[0],
-		Type: map[bool]string{true: "network", false: "client"}[*flagNet],
-	}
+	req := struct{ ID string }{args[0]}
 	var tok accesstoken.Token
 	err := client.Call(context.Background(), "/create-access-token", req, &tok)
 	if err != nil {
 		fatalln("rpc error:", err)
```

Comment:
Could you also add a TODO here about atomicity between postgres and raft db? We have the same problem that we've seen before, where a token could get created without a grant.